src/App.js

```diff
@@ -26,7 +26,7 @@ class App extends Component {
         }
       })
       // .then(data => this.setState({ ideas: data }))
-      .then(movies => this.setState({ movies: movies.movies }))
+      .then(data => this.setState({ movies: data.movies }))
```

Comment:
Good thinking making the parameter easier to understand! Was getting confusing with movies x4 🤡 😭